konfera/forms.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
```

Comment:
You should put a blank line between each group of imports (between django and konfera).

https://www.python.org/dev/peps/pep-0008/#imports